Resources/Maps/tortuga.yml

```diff
@@ -83867,7 +83863,7 @@ entities:
     parent: 33
     type: Transform
 - uid: 5436
-  type: MetempsychoticMachineCircuitboard
+  type: CloningPodMachineCircuitboard
```

Comment:
??? I already found + replaced stuff

Reply:
Ohhh, didn't see that. Just waa notified to add it.